plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -194,7 +194,7 @@ func shouldIgnore(a admission.Attributes) (bool, error) {
 // computeSecurityContext derives a valid security context while trying to avoid any changes to the given pod. I.e.
 // if there is a matching policy with the same security context as given, it will be reused. If there is no
 // matching policy the returned pod will be nil and the pspName empty.
-func (c *PodSecurityPolicyPlugin) computeSecurityContext(a admission.Attributes, pod *api.Pod, specMutationAllowed bool) (*api.Pod, string, field.ErrorList, error) {
+func (c *PodSecurityPolicyPlugin) computeSecurityContext(a admission.Attributes, pod *api.Pod, specMutationAllowed bool, validatedPSP string) (*api.Pod, string, field.ErrorList, error) {
```

Comment:
What is the goal? To speed up the validation? As @liggitt writes, this has no influence on the semantics.

Reply:
In any case I would expect a change in the godocs if the argument is relevant at all.

Reply:
> What is the goal? To speed up the validation? 

Yes. This change will speed up validataion, especially when we use remote webhook authorizor.  This can avoid uncessary http request to the authorizor.
We should avoid can [this](https://github.com/kubernetes/kubernetes/blob/b97e17603b76edf7344f49a7632b1f764d5386e6/plugin/pkg/admission/security/podsecuritypolicy/admission.go#L355) function as less as possible.

> In any case I would expect a change in the godocs if the argument is relevant at all.

Done.